src/tools/rbd_mirror/LeaderWatcher.h

```diff
@@ -98,6 +98,8 @@ class LeaderWatcher : protected librbd::Watcher {
         watcher(watcher) {
     }
 
+    virtual ~LeaderLock() = default;
```

Comment:
@tchaikov The default destructor is never generated as virtual. If you want a virtual one you have to specify it. (And the compiler was complaining about it not being virtual.)